modules/code-deployment/src/main/java/org/apache/ignite/internal/deployunit/DeploymentManagerImpl.java

```diff
@@ -226,7 +227,8 @@ public CompletableFuture<Void> undeployAsync(String id, Version version) {
                 }).thenApply(logicalTopologySnapshot -> {
                     for (ClusterNode node : logicalTopologySnapshot.nodes()) {
                         clusterService.messagingService()
-                                .invoke(node, UndeployUnitRequestImpl.builder()
+                                .invoke(node, DEPLOYMENT_UNITS,
```

Comment:
I think `DEPLOYMENT_UNITS_CHANNEL` is a better naming here. Reading the code now it is not clear that u send the message through the special channel.